_posts/2020-09-26-bring-your-own-datatypes.md

```diff
@@ -265,18 +267,19 @@ We hope this will encourage datatype researchers
   we hope this will spark interest
   in custom datatypes
   within the deep learning community.
-The Bring Your Own Datatypes framework
-  partially exists in TVM at the moment,
-  and more will be merged in (including full documentation)
-  in the coming months.
+For more documentation about the Bring Your Own Datatypes framework
+  please visit the developer tutorial [Bring Your Own Datatypes to TVM](https://tvm.apache.org/docs/tutorials/dev/bring_your_own_datatypes.html#sphx-glr-tutorials-dev-bring-your-own-datatypes-py){:target='_blank'}.
 
   
 ---
 
 *Gus Smith is a PhD student at the University of Washington working with Luis Ceze and Zachary Tatlock at the intersection of computer architecture and programming languages. His website is [justg.us](https://justg.us){:target='_blank'}.*
 
+*[Andrew Liu](https://github.com/hypercubestart){:target='_blank'} is an undergraduate student at the University of Washington and a member of UW CSE SAMPL/PLSE lab.*
```

Comment:
"SAMPL and PLSE labs", I'd say. You could link to their websites if you want, uwplse.org and https://sampl.cs.washington.edu/